src/CMakeLists.txt

```diff
@@ -132,6 +132,12 @@ set (QuokkaObjSources "${CMAKE_CURRENT_SOURCE_DIR}/main.cpp" "${CMAKE_CURRENT_SO
 #endif()
 #link_libraries(QuokkaObj)
 
+add_subdirectory(PopIII)
+
+
+add_subdirectory(StarCluster)
+#add_subdirectory(PrimordialChem)
```

Comment:
Is there a reason these need to be listed first?

Reply:
Nah, I put them first so that I didn't have to wait for other tests to finish to see if these tests would pass.